bli-guides/_how-tos/bli-ifttt-webhooks-how-to.md

```diff
@@ -31,7 +31,7 @@ Enter to the web-interface of _BeoLiving Intelligence_ and follow the next steps
 URL is _https://maker.ifttt.com/trigger/myFabulousEvent/with/key/abcDFgHijK_, insert just _https://maker.ifttt.com/trigger/_.
 
     <div class="text-center">
-      <img src="pictures/bli-ifttt-webhooks-how-to/custom-http.png" class="img-fluid" alt="Custom HTTP Connection Settings"/>
+      <img src="https://khimo.github.io/bli-guides/pictures/bli-ifttt-webhooks-how-to/custom-http.png" class="img-fluid" alt="Custom HTTP Connection Settings"/>
```

Comment:
Es necesario usar direcciones absolutas, no es mejor usar path absolutos, e.j.:

`
<img src="/bli-guides/pictures/bli-ifttt-webhooks-how-to/custom-http.png" class="img-fluid" alt="Custom HTTP Connection Settings"/>
`
De esa forma va a funcionar en cualquier dominio. Comento en este pero hay varios que los cambiaste y agregaste el domino.